packages/crud/src/vaadin-crud.js

```diff
@@ -607,10 +607,10 @@ class Crud extends SlotMixin(ControllerMixin(ElementMixin(ThemableMixin(PolymerE
 
   static get observers() {
     return [
-      '__headerNodeChanged(_headerNode, __isNew, i18n.newItem, i18n.editItem)',
-      '__formChanged(_form, theme, include, exclude)',
-      '__onI18Change(i18n, _grid)',
-      '__onEditOnClickChange(editOnClick, _grid)',
+      '__headerPropsChanged(_headerNode, __isNew, i18n.newItem, i18n.editItem)',
+      '__formPropsChanged(_form, theme, include, exclude)',
```

Comment:
**note:** The conflict was here. It turns out that V23 is still using the public `theme` property while the refactoring was done for the `master` branch where the protected `_theme` property is used.